### ISSUE TITLE
ring 0.17 is not supported on old Linux

### ISSUE THREAD

**Issue by rise0chen** [rustls/rustls#1605]
https://github.com/briansmith/ring/issues/1789

Can we  downgrade ring to 0.16 ?

**Comment by djc** [comment]
I don't think that makes sense for the rustls project. I suggest you stick with rustls 0.21.7.

**Comment by rise0chen** [comment]
This is a breaking change. ring 0.17 should be used in  rustls 0.22.x.

**Comment by djc** [comment]
It is not a breaking change for most people, similar to how MSRV bumps are not usually treated as breaking changes.
